server/README.md

```diff
@@ -1 +1,48 @@
 # Pailot Backend
+
+This is a Node.js application built with TypeScript
+
+# Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
+
+# Requirement
+
+To run this application, you need to have the following software installed on your system:
+
+Node.js (version 12 or higher)
+npm/yarn
+
+# Installation
+
+Clone the repository:
+
+git clone https://github.com/[username]/pailot-for-pi.git
```

Comment:
This should be
```bash
git clone https://github.com/pi-apps/pailot-for-pi.git
```

Also, can you add a fenced block to all the commands that needs to be run, like this one?
https://www.markdownguide.org/extended-syntax/#fenced-code-blocks

Reply:
It looks like something like this 
```
```bash
git clone https://github.com/pi-apps/pailot-for-pi.git
```
```